sklearn/datasets/_samples_generator.py

```diff
@@ -609,6 +609,19 @@ def make_regression(
     coef : ndarray of shape (n_features,) or (n_features, n_targets)
         The coefficient of the underlying linear model. It is returned only if
         coef is True.
+
+    Examples
+    --------
+    >>> from sklearn.datasets import make_regression
+    >>> X, y = make_regression(n_samples=5, n_features=2, noise=1, random_state=42)
+    >>> X
+    array([[ 0.49671415, -0.1382643 ],
```

Comment:
Seeing how `make_regression` performs a `np.dot`, I am a little concerned over slight differences between platforms. What do you think of using `...` to be more lenient and only show 4 significant figures?

```suggestion
    array([[ 0.4967..., -0.1382... ],
```

This comment applies to the rest of the docstring.

Reply:
> What do you think of using `...` to be more lenient and only show 4 significant figures?
That's true, thanks for the comment @thomasjpfan!